huxley/www/js/components/AdvisorRosterView.js

```diff
@@ -57,6 +58,18 @@ var AdvisorRosterView = React.createClass({
     Modal.setAppElement('body')
   },
 
+  componentDidMount: function() {
+    DelegateStore.addListener(function() {
+      var schoolID =  CurrentUserStore.getCurrentUser().school.id;
+      var delegates = DelegateStore.getDelegates(schoolID);
+      this.setState({
+        delegates: delegates,
+        loading: false,
```

Comment:
Don't think we need to set either loading or modal_open in this callback. What was the reason for including them?